Library/Homebrew/formula.rb

```diff
@@ -598,7 +598,16 @@ def rack
   # All currently installed prefix directories.
   # @private
   def installed_prefixes
-    rack.directory? ? rack.subdirs.sort : []
+    prefixes = rack.directory? ? rack.subdirs.sort : []
+
+    aliases.each do |alias_name|
+      rack_alias = HOMEBREW_CELLAR/alias_name
+      next unless rack_alias.directory?
+
+      prefixes += rack_alias.subdirs.sort
+    end
+
+    prefixes
```

Comment:
```suggestion
    prefixes = rack.directory? ? rack.subdirs : []

    prefixes += aliases.map do |alias_name|
      rack_alias = HOMEBREW_CELLAR/alias_name
      next unless rack_alias.directory?

      rack_alias.subdirs.sort
    end.compact

    prefixes.sort
```

This will ensure that they are all sorted.

Reply:
Would it make sense to sort by `basename` so it is ordered by versions?

Reply:
Oh: may want to include `oldname` (or `old_name`) in here too.

Reply:
`oldname` is usually handled by the migrator, but I can add it here too if that makes sense.

Reply:
Yeh, would be ideal just in case it's not been run yet.